packages/by-name/kata/kata-kernel-uvm/package.nix

```diff
@@ -59,7 +59,6 @@ let
           --replace-fail 'CONFIG_MODULE_SIG=y' 'CONFIG_MODULE_SIG=n' \
           --replace-fail '# CONFIG_DMIID is not set' 'CONFIG_DMIID=y' \
           --replace-fail '# CONFIG_TMPFS_POSIX_ACL is not set' 'CONFIG_TMPFS_POSIX_ACL=y' \
-          --replace-fail '# CONFIG_TMPFS_XATTR is not set' 'CONFIG_TMPFS_XATTR=y' \
```

Comment:
This doesn't belong in this commit, right?

Reply:
It does. It was added upstream in https://github.com/kata-containers/kata-containers/pull/10790, so --replace-fail will fail after the update as the line we are trying to replace is gone.

Reply:
But it is part of the commit named "kuberesource: increase memory limit for openssl". I get that it's needed in the PR and needed because of the update, but this should be either part of the upgrade commit (for this to be bisectable) or a separate PR, or what am I missing?

Reply:
True. <3